src/dbManager.php

```diff
@@ -23,12 +23,47 @@ public function writedb($sites, $timestamp) {
         $this->pdo->query($query);
     }
 
+    /**
```

Comment:
Maybe one for an earlier review, but I'd consider renming the class to use camel casing e.g. `DbManager` or `DatabaseManager`

Reply:
Or, looking at the rest of the class, even calling it something like `ResultsService`